### DISCUSSION TITLE
Cant join multiplayer linux

### DISCUSSION THREAD

**Discussion by Twenmod** [ppy/osu#15177] [Q&A] [answered]
Im getting the error "You are not allowed to submit a score: invalid client hash " when i try to join multiplayer or playlists

this is on the newest release of osu!lazer on linux (Pop!-OS)

**Comment by gagahpangeran** [comment, marked as answer]
Can you confirm you are playing using latest official AppImage build from github?

https://github.com/ppy/osu/releases/latest/download/osu.AppImage

**Reply by Twenmod** [reply]
Oops i had build the game via dotnet but the appimage works correctly :+1:

**Comment by peppy** [comment]
Please try again now, I think it should be better.
